crates/web/frontend/src/components/editor/editor-loading-overlay.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import { Loader } from "../ui/sonner";
+import styles from "./editor.module.css";
+
+interface Props {
+	loading: boolean;
+	loadingMessage: string;
+}
+
+export const EditorLoadingOverlay = ({ loading, loadingMessage }: Props) => {
```

Comment:
move this file to `editor/overlay/editor-loading-overlay.tsx`? or to `editor/overlay/loading.tsx`? In order to structure more the editor folder, it seems that it have too many files with different purposes